sound/soc/sof/trace.c

```diff
@@ -15,6 +15,9 @@
 #include "ops.h"
 #include "sof-utils.h"
 
+#define CREATE_TRACE_POINTS
+#include <trace/events/sof.h>
+
```

Comment:
I would add this to core.c, the trace.c does not have much to do with tracing of the kernel, it is for firmware tracing